topic-offset-service/README.md

```diff
@@ -40,7 +40,7 @@ See topic-offset-service/doc/topic-offset-service.drawio for the source of this
 (fields marked with * are acquired from headers metadata)
 
 The Topic-Offset Service monitors and watches the `masked-resource` Kafka topic and reports the offsets that satisfy some prerequisites. 
-This service accepts a `TopicOffsetRequest`, which contains a token, it is then used to subscribe to the kafka topic and check for a Start of Stream message, declared in the `StreamMarker` class. 
+This service accepts a `TopicOffsetRequest`, which contains a token value, the service will then look at the value of the `x-request-token` header to check if it is a `START` message. 
```

Comment:
```suggestion
This service retrieves the values of the `x-request-token` and `x-stream-marker` headers to decide if the `START` has been found for this token. 
```

Reply:
At the very least, the START/END comes from the `x-stream-marker` header, not the `x-request-token` header

Reply:
Updated